Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -104,12 +104,35 @@ protected virtual bool HasContentAttribute
       get { return false; }
     }
 
+    /// <summary>
+    /// Provides secure forwarding of additional attributes to  <see cref="BocColumnRenderingContext"/>
+    /// </summary>
+    public struct ParamObjectWithWriter
+    {
+      private BocColumnRenderingContext<TBocColumnDefinition> _renderingContext;
+
+      public ParamObjectWithWriter (BocColumnRenderingContext<TBocColumnDefinition> renderingContext)
+      {
+        _renderingContext = renderingContext;
+      }
+
+      public void AddAttributeToRender (string name, string value)
+      {
+        _renderingContext.Writer.AddAttribute(name, value);
+      }
+
+      public TBocColumnDefinition GetColumnDefinition ()
+      {
+        return _renderingContext.ColumnDefinition;
+      }
```

Comment:
We should convert this method into a property since `ColumnDefinition` is also a property